kordac/processors/errors/TagNotMatchedError.py

```diff
@@ -0,0 +1,15 @@
+from kordac.processors.errors import Error
+
+class TagNotMatchedError(Error):
```

Comment:
oohhh I understand, it's for any case where there is not matching start and end tag, gotcha :+1: